### DISCUSSION TITLE
emacs freezes for 5s when in sh-mode and searching for candidates

### DISCUSSION THREAD

**Discussion by RemiBraizet** [company-mode/company-mode#1477] [Q&A]
hi, i have the same problem described here
https://github.com/company-mode/company-mode/discussions/1383#discussioncomment-6852292
but i'm on Emacs27, is there anything i can do to circumvent the issue ?

**Comment by dgutov** [comment]
Perhaps you're seeing https://github.com/company-mode/company-mode/issues/1351.

Are you using company-mode from ELPA or from MELPA? The fixed version is currently published to the latter (and to ELPA-devel, FWIW).

**Reply by RemiBraizet** [reply]
quite late, but here am i
i did a clean install from melpa, and i dont get the problem anymore
thanks
